src/back/auth/src/repo/table_state.cpp

```diff
@@ -1,30 +1,16 @@
-#include "repository.hpp"
+#include "table_state.hpp"
 
 #include "userver/components/component_config.hpp"
 #include "userver/components/component_context.hpp"
 #include <userver/yaml_config/merge_schemas.hpp>
 #include <userver/storages/postgres/component.hpp>
 
-namespace svetit::auth {
+namespace svetit::auth::table {
 
-/*static*/ yaml_config::Schema Repository::GetStaticConfigSchema()
-{
-	return yaml_config::MergeSchemas<components::LoggableComponentBase>(R"(
-type: object
-description: Main repository component
-additionalProperties: false
-properties:
-  some-var:
-    type: string
-    description: some desc
-)");
-}
-
-Repository::Repository(
-		const components::ComponentConfig& conf,
-		const components::ComponentContext& ctx)
-	: components::LoggableComponentBase{conf, ctx}
-	, _pg{ctx.FindComponent<components::Postgres>("database").GetCluster()}
+State::State(
+	    storages::postgres::ClusterPtr pg
+        )
+	: _pg{pg}
```

Comment:
move тут нужен чтобы объект меньшее кол-во раз скопировался.
```suggestion
State::State(storages::postgres::ClusterPtr pg)
	: _pg{std::move(pg)}
```

Reply:
вот это не понял. предлагаю обсудить, и тогда внесу правки везде, где это нужно.

Reply:
поправил